publiccode.yml

```
@@ -0,0 +1,89 @@
+releaseDate: '2024-09-17'# needs to be updated
+softwareType: library
+logo: "https://github.com/iTowns/itowns/blob/master/examples/images/itowns_logo.png"
+inputTypes: # ToDo : complete
+  - application/json
+platforms:
+  - web
+categories:
+  - website-builder
+  - data-visualization
+  - it-development
+usedBy: # ToDo : complete and ask for permissions
+  - IGN (https://www.ign.fr/)
+  - Ciril Grp (https://www.ciril.net/fr/)
+  - Sogelink (https://www.sogelink.com/)
+  - Laboratoire Liris (https://liris.cnrs.fr/)
+  - Laboratoire Lastig (https://www.umr-lastig.fr/)
+roadmap: "https://www.itowns-project.org/"
+developmentStatus: stable
+intendedAudience:
+  scope:
+    - science-and-technology
+    - research
+    - environment
+    - government
+    - defense
+description:
+  en:
+    documentation: 'https://www.itowns-project.org/'
+    apiDocumentation: 'https://www.itowns-project.org/itowns/docs/#tutorials/Fundamentals'
+    shortDescription: open-source framework designed for web-based visualisation, navigation and interaction with 2D and 3D geospatial data.
+    longDescription: |
+        iTowns is an open-source framework designed for web-based visualisation, navigation and interaction with 2D and 3D geospatial data on globes and maps. Built on Open Geospatial Consortium (OGC) open standards, it is developed with data and service interoperability in mind. It seamlessly integrates with geographical services, offering support of standard raster and vector data, including aerial imagery and terrain models. The framework supports large, heterogeneous 3D datasets such as OGC's 3D Tiles, making it ideal to build application for urban-planning and environmental monitoring. It can be easily extended to support other open formats, offering a highly customizable platform for developers.
+ 
+        iTowns is a geographic commons, developed collectively by a diverse community of contributors, comprising independent developers, public organizations, research laboratories and private companies. iTowns offers a modular framework to build a wide range of use cases, including visualisation, GIS, environmental and educational applications, making it versatile and adaptable for different geospatial projects.
+    features: # ToDo : complete ?
+      - Navigation and interaction with 2D and 3D geospatial data
+      - Built on Open Geospatial Consortium (OGC) open standards (wms, wfs, copc, 3dTiles, ...)
+    screenshots: # ToDo : complete ?
+      - 'https://raw.githubusercontent.com/iTowns/itowns.github.io/master/images/itownsReleaseXS.jpg'
+      - 'https://camo.githubusercontent.com/337a7f2d1c254e9099b30dc336982518fd6e8c8733c1f29fe669290b05f33905/687474703a2f2f7777772e69746f776e732d70726f6a6563742e6f72672f696d616765732f6d6f6e746167652e6a7067'
+      - 'https://raw.githubusercontent.com/iTowns/itowns/refs/heads/master/docs/tutorials/images/3DTiles-mesh-b3dm-2.png'
+    genericName: iTowns
+legal:
+  license: MIT and CeCILL-B
```

According to SPDX specification
> When there is a choice between licenses (“disjunctive license”), they should be
separated with “OR”. If presented with a choice between two or more licenses, use the disjunctive binary “OR” operator to construct a new license expression.

```suggestion
  license: MIT OR CECILL-B
```